src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +52,49 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        MovieSession todayMovieSession = new MovieSession();
+        todayMovieSession.setCinemaHall(firstCinemaHall);
+        todayMovieSession.setMovie(fastAndFurious);
+        todayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector.getInstance(
+                MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(todayMovieSession);
 
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
+        System.out.println(movieSessionService.get(todayMovieSession.getId()));
```

Comment:
Consider replacing `System.out.println` with a logging statement for consistency with the rest of the code, which uses the `Logger` class.